### PR TITLE
Make sure env is an empty object (not undefined)

### PR DESCRIPTION
**Pull request by Baggz**

### CODE REVIEW

File: lib/commonmark.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ function renderHtml(markdownString, options, cb) {
 }
 
 function parse(markdownString, cb) {
-  return cb(null, commonmarkRenderer.parse(markdownString));
+  return cb(null, commonmarkRenderer.parse(markdownString, {}));
```
Comment (kuba-kubula) [inline lib/commonmark.js]: please first do a try/catch with retrieving the results of the `.parse` method; to be able to call callback with an Error, or with `null, results`

**Comment by XVincentX** [comment]
Yes, I just noticed it. Will fix.

**Comment by kuba-kubula** [comment]
Btw - can we add an option to pass the `env` too? Might be usable in future.

File: lib/commonmark.js (hunk 1 above)
Comment (XVincentX) [inline lib/commonmark.js]: Ok for the first - I do not understand the second statement.
It *is* indeed calling `parse` directly - with the default apib options.

Comment (kuba-kubula) [inline lib/commonmark.js]: It is calling it, but it is not a synchronous API call. => it doesn't make sense to be asynchronous-looking-like

Comment (XVincentX) [inline lib/commonmark.js]: image

Comment (XVincentX) [inline lib/commonmark.js]: Should I change the interface and make a breaking change?
Even if I would change the name it would be a breaking change anyway.

Comment (kuba-kubula) [inline lib/commonmark.js]: maybe introduce `parseSync` and/or `parseAsync`.

**Pull request merged**